web-server/src/types/openapi.yaml

```diff
@@ -379,3 +391,16 @@ components:
               type: integer
             validatorId:
               type: integer
+    UserError:
+      type: object
+      properties:
+        raw:
+          type: object
```

Comment:
I meant that you should remove this property entirely because it's a security risk.